### ISSUE TITLE
Bookmarks not shown for a particular PDF

### ISSUE THREAD

**Issue by user1823** [sumatrapdfreader/sumatrapdf#4192]
**SumatraPDF version**
- v3.6.16022 64-bit

**Describe the bug**
SumatraPDF doesn't show any bookmarks for the following PDF file, but Chrome does.

**File that reproduces the problem**
[David Freedman_ Robert Pisani_ Roger Purves - Statistics.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/15013462/David.Freedman_.Robert.Pisani_.Roger.Purves.-.Statistics.pdf)

**Screenshots**
<img width=300 src="https://github.com/sumatrapdfreader/sumatrapdf/assets/92206575/0aaac462-4b32-4ddd-85ea-8bfa78804fb0">
<img width=300 src="https://github.com/sumatrapdfreader/sumatrapdf/assets/92206575/82acfa63-13d6-4b7d-a644-654bb3e4f8f4">

**Comment by kjk** [comment]
According to mupdf this pdf file is malformed:
```
Bad or missing prev pointer in outline tree, repairing
format error: Outline last pointer still bad or missing despite repair
Couldn't load outline for 'C:\Users\kjk\Downloads\David.Freedman_.Robert.Pisani_.Roger.Purves.-.Statistics.pdf'
```

**Comment by GitHubRulesOK** [comment]
Hmm in such cases I try different tricks to fix but Acrobat Reader and Pdfium Edge will often reshuffle contents but the bookmarks are still the same even though one file is bigger and one smaller.

The best of the Lot is MuPDF and again it sees bookmarks and offers a fix but I have to fiddle with settings to ensure it alters to include the bookmarks.

[Uploading David.Freedman_.Robert.Pisani_.Roger.Purves.-.Statistics - Copya.pdf…]()


We can see the signature is now Mupdf compatible too (%ÂµÂ¶)

**Comment by user1823** [comment]
> According to mupdf this pdf file is malformed:

But, MuPDF-GL (v1.23.0) displays the bookmarks correctly. So, I think that the issue is in SumatraPDF.

(I am not saying that the PDF doesn't have any issues. But, if MuPDF can display it correctly, SumatraPDF should also do the same.)

<img width=300 src="https://github.com/sumatrapdfreader/sumatrapdf/assets/92206575/64a73c72-4de5-4d2a-ada6-daff4cb9d765">

> The best of the Lot is MuPDF-GL and again it sees the bookmarks as working yet offers a fix, but I have to fiddle with settings to ensure it alters to include the bookmarks.

@GitHubRulesOK, how did you use MuPDF-GL to repair the PDF? I know about `mutool clean`. Are you talking about that?

**Comment by kjk** [comment]
It doesn't work in latest mupdf, I opened an issue https://bugs.ghostscript.com/show_bug.cgi?id=707750

**Comment by GitHubRulesOK** [comment]
No MuPDF viewer is an format editor you can do many things such as draw annotation with or without incremental save or in this case resave with corrected syntax. similar to command line clean but inside a gui.

**Comment by user1823** [comment]
> resave with corrected syntax. similar to command line clean but inside a gui.

Exactly how? I couldn't find anything reference online.

**Comment by GitHubRulesOK** [comment]
@user1823 
press A bottom right "save as pdf" > deselect incremental and then the cleanup commands should be at the bottom 
Save as must be a new name and needs click not enter !!

**Comment by chrisjj** [comment]
"Bad or missing prev pointer in outline tree, repairing"

Could someone explain where/what this "outline tree" is? I see nothing named that in the UI or Help.

**Comment by GitHubRulesOK** [comment]
@chrisjj 
The Adobe official name for the external list of contents is called an "Outline" of contents which are random PDF objects that name pages and are displayed as whole "Book" marks commonly call in SumatraPDF and Others as ToC, BookMarks [sidebar] navigator or show as a TreeView

**Comment by chrisjj** [comment]
Thanks. So this outline tree in SumatraPDF is in the Bookmarks pane?

**Comment by GitHubRulesOK** [comment]
yes I see you wish to show all names covered in page window but that is not easy PAGES do not contain their names it is the tree has next name and just current page Thus when focus is in Bookmarks easy to show a page but easy other way round to say which marks are in future or past for that page.

**Comment by GitHubRulesOK** [comment]
Included in recent round of mupdf fixes.

<img width="795" height="417" alt="Image" src="https://github.com/user-attachments/assets/306d0d68-cfe2-48e0-8eaf-c75498aba385" />
